installer/templates/new/web/views/error_helpers.ex

```diff
@@ -26,10 +26,9 @@ defmodule <%= application_module %>.ErrorHelpers do
     #
     #     dngettext "errors", "1 file", "%{count} files", count
     #
-    Gettext.dngettext(<%= application_module %>.Gettext, "errors", msg, msg, opts[:count], opts)
-  end
-
-  def translate_error(msg) do
-    Gettext.dgettext(<%= application_module %>.Gettext, "errors", msg)
+    case Keyword.has_key?(opts, :count) do
```

Comment:
:+1: for the `if`!
re: the comment, it seems to apply whether we have `:count` defined or not. Maybe I can add another example without count and amend the comment to clarify what's going on with `:count`?